CONTRIBUTING.md

```diff
@@ -11,4 +13,48 @@ Although contributing to the project is allowed and even encouraged it has to fo
 
 
 
-This is a *guideline* which means that some exceptions may be made. However it is recommend that you respect it for your contribution to be validated. If you have any other question or think a point is unclear or too limiting feel free to contact an @admin
\ No newline at end of file
+This is a *guideline* which means that some exceptions may be made. However it is recommend that you respect it for your contribution to be validated. If you have any other question or think a point is unclear or too limiting feel free to contact an @admin
+
+## Installing an editing
+
+### Requirements
+
+You will need python 3 and the library `discord.py`.
+
+*(We would also recommend to work on Linux :penguin:)*
+
+### The serious stuff
+
+Fork the repository from the [GitHub page](https://github.com/s0lst1ce/ForeBot). This will make a copy of the repo on your account.
+
+Then, on your computer, go to the directory where you want to install the bot and do this command:
+
+`git clone https://github.com/s0lst1ce/ForeBot.git`
+
+Then:
+
+`cd ForeBot` 
+
+And there you go! This is a fast [explanation](https://discordapp.com/developers/docs/intro#bots-and-apps) about how to "create" a bot form the official discord's doc. You have to make an "app" and then you'll have a token in the "Bot" section of your App you will be able to see/copy your token.
+
+You'll have to copy this token to put it in the file `settings.py`. **Warning! Make sure to don't commit your token, remove it before!** If you accidentally commit it you can regenerate it.
+
+You'll also have to invite the bot on your developing server.
+
+To run the bot you just have to execute this command in your shell: 
+
+`python executioner.py` (depending on your installation you'll maybe have to use the command `python3`)
+
+You're now able to edit the Bot! :tada: :confetti_ball:
+
+### Pull Requesting!
+
+Now that you have improve our bot you have to publish what you did! For that you have to make a Pull Request (PR), it's like a proposition.
+
+Please respect the [Rules](#Rules) and then your improvement will be quickly accepted! :smile:
+
+You have to commit what you've done to your forked repository (on **your** GitHub account), and  then when it's done to push them.
+
+After that you have to make a PR, for that you can use a client (e.g. GitKraken) or do it directly from the GitHub website. Go on the forked repo (e.g. https://github.com/YOUR_NAME/ForeBot) and then go to the "Pull Requests" tab (between Code and Projects). Now click on "New Pull Request", select `s0lst1ce/ForeBot` for the "base repository" and your repo for the "head repository". *For both ones, select the right branch.*
+
+Then you can submit your PR, after that we will review your PR to avoid conflicts. It may takes few days, be patient, but you will certainly be in touch threw the GitHub's messages: :wink:
```

Comment:
*but we'll always seriously consider any subimission :wink:*

Reply:
Instead of * but you will certainly be in touch threw the GitHub's messages: :wink:*

Reply:
Maybe also add a *please* before *be patient*?

Reply:
> Maybe also add a _please_ before _be patient_?

I don't think this is important, I think they know that :p

Reply:
I've found excessive politeness favorable to normal one since we cannot know how it will be interpreted on the other end. We don't all think the same way.
Now it's your PR so I leave you the final word on it.

Reply:
I find that a bit to formal, but it's your bot ^^ I don't know how you want people to see it.

Reply:
and line 60 there is : `It may takes few days, be patient` I can add "please" if you want.

Reply:
*take* and do what you want for the "please". You're the boss ;)